bootstrap/package.json

```diff
@@ -3,7 +3,9 @@
     "version": "0.1.0",
     "private": true,
     "dependencies": {
-        "@gooddata/gooddata-js": "^12.5.0",
+        "@gooddata/sdk-backend-bear": "^8.0.0-alpha.72",
```

Comment:
The `@gooddata/sdk-backend-tiger` is probably missing here.

Reply:
`@gooddata/sdk-backend-tiger` will generate when the user specifically specifies it in the CLI before creating the app.